index.md

```diff
@@ -15,7 +15,7 @@ Beemo is a free antispam Discord Bot that can stop userbot raids against your se
 
 ## Important Documentation Notes
 
-**For all commands,** Beemo does not currently use Discord's slash commands. This means that if another bot is using the slash command `/help`, you cannot type `/help` to use for Beemo. To fix this issue temporarily while slash command support is being developed for Beemo, simply add a space in front of any command that might be taken by another bot's slash commands. Example: ```(space)/help```
+**If you can't see the commands of XP, Honey or Mod**, then you most likely have the feature disabled. Beemo removes the commands from your server when it finds that the feature is not enabled on your server. To enable the feature, you can use the `/toggle` command.
```

Comment:
Do you think adding something like:
> **Moderation, XP, and Honey are features of Beemo Max. You may not be able to enable, or use them unless you are a Beemo Max subscriber.**?

Reply:
Without the comma after "enable", but yeah that sound good. The first "Beemo Max" could also link to the subscription page.

Reply:
Should we add it right above the highlighted line, below, or in the highlighted line? (the line highlighted in this comment)

Reply:
Probably above, otherwise it would first say that you should enable the feature, and then say you can't enable the feature. Better lead with what you can't do in the first place.